allensdk/brain_observatory/ecephys/current_source_density/_interpolation_utils.py

```diff
@@ -0,0 +1,175 @@
+from typing import Tuple
+
+import numpy as np
+
+from scipy.interpolate import RegularGridInterpolator, griddata
+
+
+def regular_grid_extractor_factory(timestamps: np.ndarray,
```

Comment:
No longer used right?

Reply:
Still used for extracting lfp windows centered around events. Because the LFP timestamps may not 'exactly' match up with event timestamps a little bit of interpolation is still needed.